.env.example

```diff
@@ -14,3 +14,7 @@ DEFAULT_KEY_TYPE=local
 # This symmetrical key is used to encrypt private keys
 # that are stored in the database.
 ENCRYPTION_KEY=faae4ed1c30f4e4555ee3a71f1044a8e
+
+ENABLED_TOKENS=FUSD:0xf8d6e0586b0a20c7:fusd,FlowToken:0x0ae53cb6e3f42a79:flowToken
```

Comment:
`ENABLED_TOKENS` was not included in the example env file, was a common question how to set it from builders.